src/sprite.c

```diff
@@ -354,7 +392,7 @@ s32 func_802DE748(s32 arg0, s32 arg1) {
         return -1;
     }
 
-    return componentList[arg1]->unk_4F;
+    return (u8)componentList[arg1]->unk_4C;
```

Comment:
why was this changed?

Reply:
actually, maybe it's correct. I'm just wondering if there was something to corroborate it

Reply:
ohhhh it's the ```
if (arg2 != 0) {
                    component->unk_4C |= 0x10000000;
                } else {
                    component->unk_4C &= ~0xF0000000;
                }
                ```
                I think this indicates it's actually a bitfield or something, but we can worry about that later. Though, instead of the u8 casts, we should probably be doing & 0xFF on that field.

Reply:
(there's another u8 cast or two on that field elsewhere in this PR, so this applies to all of those)

Reply:
changed to `& 0xFF`